lib/core/src/Cardano/Wallet/Primitive/Model.hs

```diff
@@ -354,14 +353,44 @@ totalUTxO pending (Wallet u _ s) =
     entriesToExcludeForTx :: Tx -> Set TxIn
     entriesToExcludeForTx tx = Set.fromList $ fst <$> tx ^. #resolvedInputs
 
+-- | Retrieve the change 'UTxO' contained in a set of pending transactions.
+--
+-- We perform /some/ address discovery within the list of pending addresses,
+-- but we do not store the result.
+-- Instead, we essentially assume that the address discovery state @s@ contains
+-- enough information to collect the change addresses in the pending
+-- transactions.
+--
+-- Cave:
```

Comment:
what do you mean by 'Cave' - limitations?

Reply:
Yes, I meant the latin "beware!". Rodney changed it to "Caveats", which is better.